packages/lesswrong/lib/collections/posts/views.ts

```diff
@@ -1518,7 +1518,9 @@ Posts.addView("myBookmarkedPosts", (terms: PostsViewTerms, _, context?: Resolver
   
   return {
     selector: {
-      _id: {$in: bookmarkedPostIds}
+      _id: {$in: bookmarkedPostIds},
+      isEvent: viewFieldAllowAny,
+      groupId: null
```

Comment:
@b0b3rt @darkruby501 This is the other change that will affect LW. It makes it so that bookmarked posts that have a `groupId` and bookmarked events appear in your bookmarks list. Previously, those were filtered out. I assume that was a bug.